driveteam.py

```diff
@@ -0,0 +1,59 @@
+import wpilib
+
+class DriveTeam:
+
+    pilot = wpilib.XboxController(0)
+
+    copilot = wpilib.XboxController(1)
+
+    def get_strafe_commands(self): 
+        return self.pilot.getLeftX() 
+
+    def get_drive_commands(self): #translational(foward/backward)
+        return self.pilot.getLeftY()
+
+    def get_turn_commands(self): #left/right
+        return self.pilot.getRightX()
+    
+    def get_winch_commands(self): #deploy winch
+        return self.pilot.getStartButton() 
+    
+    def get_hook_extension_commands(self): 
```

Comment:
Also, i noticed that all of these method names end in "commands" as a plural but they only return one thing. I am not 100% familiar with the  nature of `wpilib` and what we are returning, but its best practice to be explicit with method naming.